### PR TITLE
Script for coverage when AutoCompleteList have on of invalid param

### PR DESCRIPTION
**Pull request by Aleksandriys**
@aarotar @OVikhrov please review.

### CODE REVIEW

File: test_scripts/API/AutoCompleteList/014_ATF_SetGlobalProperties_N_AutoCompleteList_InvalidParam_GENERIC_ERROR.lua [hunk 1]
```diff
@@ -0,0 +1,99 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = false, resultCode = "GENERIC_ERROR"> to mobile app
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/014_ATF_SetGlobalProperties_N_AutoCompleteList_InvalidParam_GENERIC_ERROR.lua]: @Aleksandriys, remove duplicated "SDL" in requirement summary

File: test_scripts/API/AutoCompleteList/014_ATF_SetGlobalProperties_N_AutoCompleteList_InvalidParam_GENERIC_ERROR.lua [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param param to HMI
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/014_ATF_SetGlobalProperties_N_AutoCompleteList_InvalidParam_GENERIC_ERROR.lua]: @Aleksandriys,
1. Description currently looks like "expected result". Please try to use description describing case: "Case when ..."
2. please update description with particular resultCode from HMI (please specify "GENERIC_ERROR") and not <resultCode_received_from _HMI>

File: test_scripts/API/AutoCompleteList/014_ATF_SetGlobalProperties_N_AutoCompleteList_InvalidParam_GENERIC_ERROR.lua [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList> 
+-- 3. HMI send responce on SDL with invalid value of valid param and other valid params with valid values.
+-- 4. SDL respond <success = false, resultCode = "GENERIC_ERROR"> to mobile app
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/014_ATF_SetGlobalProperties_N_AutoCompleteList_InvalidParam_GENERIC_ERROR.lua]: @Aleksandriys, looks like you missed "Expected result"(mandatory in template) in header.
I guess, at least p3."SDL respond with <success = false, resultCode = "GENERIC_ERROR">  to mobile app" Should be in expected result.

File: test_scripts/API/AutoCompleteList/014_ATF_SetGlobalProperties_N_AutoCompleteList_InvalidParam_GENERIC_ERROR.lua [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+   end
+end)
+end
+
+--[[ Postconditions ]]
+commonFunctions:newTestCasesGroup("Postconditions")
+function Test.Postcondition_SDLStop()
+  StopSDL()
+end
+
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/014_ATF_SetGlobalProperties_N_AutoCompleteList_InvalidParam_GENERIC_ERROR.lua]: @Aleksandriys, please avoid redundant empty lines in  script

File: test_scripts/API/AutoCompleteList/014_ATF_SetGlobalProperties_N_AutoCompleteList_InvalidParam_GENERIC_ERROR.lua [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+          {
+            "a"
+          },
+           language = "EN-US",
+           autoCompleteList = 5
+       }
+    })
+    end)
+ --mobile side: expect response
+  :Do(function(_,data)
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/014_ATF_SetGlobalProperties_N_AutoCompleteList_InvalidParam_GENERIC_ERROR.lua]: @Aleksandriys, no need of **:Do function** there, 
EXPECT_RESPONSE(cid, {success = false, resultCode = "GENERIC_ERROR"}) 
is enough

**Comment by Aleksandriys** [comment]
@aarotar please review.

**Pull request merged**